hello-core-logic/createBoardMimicTrello.js

```diff
@@ -0,0 +1,34 @@
+// Write a function named createBoard that takes one argument, boardName, then creates a board if there is no board with
+// the same name (meaning that it adds a board to the object assigned to the variable, hello.) The function should return,
+// Board '<board-name>' was created. If there is already a board with the same name, it should return Board already exists.
+// Use your listBoards function to display before and after creating a board to test that it worked successfully.
+
+const hello = {
+    'Tester Board': {
+      'To Do': ['Laundry', 'Buy Apples', 'Pay Phone Bill'],
+      'Doing': ['Laundry', 'Studying Javascript', 'Studying HTML', 'Studying Ruby'],
+      'Done': ['Laundry']
+    },
+    'Dreams': {
+      'Wish List': ['Conquer the Seven Kingdoms', 'Get my baby back', 'My hand needs to chill']
+    }
+};
+
+board = "Cuisini";
+
+function createBoard(obj, boardName) {
+  let arr = Object.keys(obj);
+  arr.forEach(function(key) {
+    if (key === boardName) {
+      message = `Board '${boardName}' already exists, so it cannot be added!`;
+      return message;
+    } else {
+      obj[boardName] = [];
+      message = `Board '${boardName}' was created!`;
+    }
+  });
+
+  return message;
+  }
+
```

Comment:
When creating a board, to check if the board already exists, all you need to do is the following:
```javascript
if (hello[boardName]) {
   return `Board '${boardName}' already exists`;
}
```

This works because javascript is a 'truthy' language. This means that a statement/variable does not need to explicitly return `true` in order for an if statement to be true. It will be true as long as it is not `false`, `undefined`, `null` , or `0`.

So, if there is a board with that name, hello[boardName] will return an object that represents that board, and since an object is not one of the things listed above, the statement will be `true` for the if statement.

From there, you can have an else statement that looks like the following:

```javascript
else {
   hello[boardName] = {};
   return `Board '${boardName}' was created!`;
}
```

There is no need for the loop at all in this solution, making it much simpler.